src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -357,9 +594,26 @@
 ;; Data Browser
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(re-frame/reg-event-fx
```

Comment:
this is a lot of overhead to do EVERY time. The normal page timeout with logout redirect should handle this.

Reply:
This effect is only called during statement browser + actor delete operations, so I'm not sure what you mean by "EVERY time."